.github/workflows/ci_cron_daily.yml

```diff
@@ -0,0 +1,81 @@
+name: Daily Cron Tests
+
+on:
+  schedule:
+    # run at 6am UTC on Tue-Fri (complete tests are run every Monday)
+    - cron: '0 6 * * 2-5'
+  pull_request:
+    # We also want this workflow triggered if the 'Daily CI' label is added
+    # or present when PR is updated
+    types:
+      - synchronize
+      - labeled
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  TOXARGS: '-v'
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    if: (github.repository == 'spacetelescope/slitlessutils' && (github.event_name == 'schedule' || github.event_name == 'push' || github.event_name == 'workflow_dispatch' || contains(github.event.pull_request.labels.*.name, 'Daily CI')))
+    name: ${{ matrix.prefix }} ${{ matrix.os }}, ${{ matrix.tox_env }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            python: '3.11'
+            tox_env: 'linkcheck'
+            allow_failure: false
+            prefix: ''
+
+          - os: ubuntu-latest
+            python: '3.11'
+            tox_env: 'py311-test-devdeps'
```

Comment:
Do you really need two flavors of devdeps here?

Reply:
One specifically tests Python 3.12-dev.

Reply:
I still don't see why you need to separate it out like this, but ok.

Reply:
I didn't find any Python 3.12 issues, so perhaps I'll just use py 3.12 for the devdeps.  Is that what you are using?

Reply:
I switched to one devdeps with Python 3.12.